rcl/QUALITY_DECLARATION.md

```diff
@@ -101,7 +101,7 @@ Currently nightly test results can be seen here:
 
 ### Public API Testing [4.ii]
 
-Most of the functionality of the declared API in this package is covered in its unit tests. Currently it has a line coverage of [77%](https://ci.ros2.org/job/ci_linux_coverage/85/cobertura/src_ros2_rcl_rcl_src_rcl/).
+Most of the functionality of the declared API in this package is covered in its unit tests. Currently it has a line coverage of [80%](https://ci.ros2.org/job/ci_linux_coverage/lastBuild/cobertura/src_ros2_rcl_rcl_src_rcl/).
```

Comment:
```suggestion
Most of the functionality of the declared API in this package is covered in its unit tests. Currently it has a line coverage of [80%](https://ci.ros2.org/job/ci_linux_coverage/lastSuccessfulBuild/cobertura/src_ros2_rcl_rcl_src_rcl/).
```

This way current builds or failing builds don't interfere with the link. Change in the coverage section and links in other QDs as well.

Reply:
I agree. Something that will remain true after the next PR would be good. Either '> 80%' or 'not yet 95%'